### ISSUE TITLE
Log file changes due to virtual clusters are wrong

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#399]
Originally reported of Google Code with ID 343.

```
Commit 3329f4de changed the Haskell log file from constants to functions, but introduced a bug: it uses now the daemon name instead of the correct log file, which means "ganeti-confd.log" instead of "conf-daemon.log".

This is a trivial issue, but filling it so as to not forget before 2.7 release, since it's a regression from 2.6 behaviour.
```
Originally added on 2013-01-11 12:02:17 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Actually, this is not a problem just with the Haskell code, it might be that the Python code is affected as well:

>>> pathutils.GetLogFilename(constants.MASTERD)
'/var/log/ganeti/ganeti-masterd.log'

Which is different from 2.6 as well. However, the daemon.py doesn't use that, but instead has a hardcoded list of filenames.

I suspect the whole daemon-log-files-changing is quite buggy and needs to be fixed for 2.7.
```
Originally added on 2013-01-11 12:08:06 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-01-11 12:08:29 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed and pushed to devel-2.7, marking as fixed.
```
Originally added on 2013-02-05 09:04:42 +0000 UTC.

Changed State: Released
